idc/settings.py

```diff
@@ -302,6 +302,7 @@ def GET_BQ_COHORT_SETTINGS():
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'request_logging.middleware.LoggingMiddleware',
     'offline.middleware.OfflineMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
 ]
```

Comment:
This isn't necessary--SessionMiddleware is already enabled at the top of the list.